.circleci/config.yml

```diff
@@ -75,6 +94,14 @@ workflows:
               # the build job on all tags. By default, Circle CI only kicks off
               # builds on tags if a filter is defined.
               only: /^(?:[0-9]+)\.(?:[0-9]+)\.(?:[0-9]+)(?:(\-rc[0-9]+)?)$/
+      - deploy-macos:
```

Comment:
Is there a way for us to only release if the other artefacts have been generated properly?

Reply:
Can we do:

```
requires:
  - build
  - deploy
```
??

Reply:
We can have it be a three part pipeline I think:

- build
- create-artifacts across all 3 platforms (windows, mac os x, etc)
- publish

Just not sure how to pass assets from creating the artifacts to publish.

Reply:
fyi circleci doesn't have windows machines

Reply:
Can you open a new issue for us to tackle this in future?

Reply:
Also to do with your original point. I can't really find any way of storing artifacts. While it's a nice to have I don't know if it's critical right now. I can open an issue about it to look into it in the future.